lib/Util.js

```diff
@@ -0,0 +1,56 @@
+class ArrayUtil {
+
+    static Reducer( arr, fn ) {
+        return arr.reduce( ( memo, item, i ) => ( { ...memo, ...fn( item, i ) } ), { } )
+    }
+
+    static ShuffleArray( arr ) {
+        const rv = Array.from( arr )
+       
+        rv.forEach( ( item, i ) => {
+            if( i === rv.length - 1 ) return 
+            const int = this.getRandomInclusiveInteger( i, rv.length - 1 ),
+                holder = rv[ i ]
+
+            rv[i] = rv[int]
+            rv[int] = holder
+        } )
+
+        return rv
+    }
+
+}
+
+class NumUtil {
+
+    static GetIntRange( int ) {
+        return Array.from( Array( int ).keys() )
+    }
+
+    static GetRandomInclusiveInteger( min, max ) {
```

Comment:
Might as well remove randomizers to some test util unless you can think of a reason the front end needs it